web3/package.json

```diff
@@ -30,6 +30,6 @@
     "eslint": "^8.5.0",
     "prettier": "^2.2.1"
   },
-  "version": "1.11.0",
+  "version": "1.12.0-rc1",
```

Comment:
we should remove the special tag handling in web3 CI:
- web3-docker-publish.yml
- web3-indexer-docker-publish.yml

eg:

- https://github.com/godwokenrises/godwoken/blob/develop/.github/workflows/web3-docker-publish.yml#L69-L74
- https://github.com/godwokenrises/godwoken/blob/develop/.github/workflows/web3-indexer-docker-publish.yml#L9